ngrinder-controller/src/main/resources/ngrinder_datachange_logfile/db.changelog_schema_H2.xml

```diff
@@ -73,6 +73,15 @@
 		        last_progress_message varchar(2048),
 		        mean_test_time double,
 		        peak_tps double,
+				errorRate double,
```

Comment:
This code is not backwards compatible. add new `db.changelog_schema_xx.xml` [here](https://github.com/naver/ngrinder/tree/81d57b136eac646a400aa0a90149e29199885f51/ngrinder-controller/src/main/resources/ngrinder_datachange_logfile)

Reply:
Got it,  I will fix it.